src/test/integration.tags

```diff
@@ -3,6 +3,7 @@
 @applications
 @applications.boxes
 @applications.verified
+@applications.boxes_indexer_confirmed
```

Comment:
same tag questions as in the other SDK's

Reply:
this one goes